torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -52,66 +52,20 @@ bool isBroadcasting(Node* node) {
   return broadcasting.count(node->kind());
 }
 
-// First iterate over the 'from' tensor sizes. Ignore all leading and trailing
-// dimensions that are simply one, since they can be trivially broadcasted.
-// When iterating over the dimension sizes (with reduced 'from' tensor),
-// starting at the trailing dimension, the dimension sizes must either be equal,
-// or one of them does not exist. If a broadcast candidate is not found at the
-// trailing dimension, search at the leading dimension. If one is found here,
-// return the `axis` argument to be emitted to ONNX on the broadcasting operator
-//
-// Note that this is NOT equivalent to numpy broadcasting semantics, and do
-// not represent the generalized broadcasting that Pytorch implements.
-// Rather, this is Caffe2-style broadcasting.
-//
-// Return value is 1) Whether this expand is fusable, 2) the `axis` argument we
-// should emit to ONNX. Coming from a Pytorch frontend, this should either not
-// be emitted (if we're broadcasting trailing dimensions) or it should be
-// emitted as `0` (leading dimensions.)
```

Comment:
The comment is gone... so what are the semantics now, and why are they this way?

Reply:
++ will add comment